manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Codilime.Veles
+PackageVersion: 2018.05.0.TIF

```

```suggestion
PackageVersion: 2018.05
```
The PackageVersion should match what is written to control panel, or `AppsAndFeaturesEntries` need to be specified. After changing this value, the version folder the manifest is in will need to be renamed